setup.py

```diff
@@ -16,29 +16,36 @@
 #   TORCH_XLA_PACKAGE_NAME
 #     change the package name to something other than 'torch_xla'
 #
-#   COMPILE_PARALLEL=1
-#     enable parallel compile
-#
-#   BUILD_CPP_TESTS=1
-#     build the C++ tests
-#
-#   XLA_DEBUG=0
-#     build the xla/xrt client in debug mode
-#
-#   XLA_BAZEL_VERBOSE=0
+#   BAZEL_VERBOSE=0
 #     turn on verbose messages during the bazel build of the xla/xrt client
 #
 #   XLA_CUDA=0
 #     build the xla/xrt client with CUDA enabled
 #
+#   XLA_CPU_USE_ACL=0
+#     whether to use ACL
+#
 #   BUNDLE_LIBTPU=0
 #     include libtpu in final wheel
 #
+#   GCLOUD_SERVICE_KEY_FILE=''
+#     file containing the auth tokens for remote cache/build. implies remote cache.
+#
+#   BAZEL_REMOTE_CACHE=""
+#     whether to use remote cache for builds
+#
+#   TPUVM_MODE=0
+#     whether to build for TPU
+#
+#   SILO_NAME=""
+#     name of the remote build cache silo
```

Comment:
can you explain more about what's "remote build cache silo"?

Reply:
Builds and tests that use the remote cache are siloed within the cache with a cache silo key. Each uniquely-configured machine running Bazel should have a unique cache silo key.

Reply:
When running from scripts (ci/test), it is autoconfigured. The default is to use a common silo, so maybe a bit poorer cache performance but still great. I've seen docker digest being used as the key in some places.